app/v1/services/UnignoreEmploymentService.scala

```diff
@@ -50,12 +50,24 @@ class UnignoreEmploymentService @Inject() (connector: UnignoreEmploymentConnecto
       "SERVICE_UNAVAILABLE"       -> InternalError
     )
 
-    val extraTysErrors = Map(
+    val extraIfsTysErrors = Map(
       "INVALID_CORRELATION_ID" -> InternalError,
       "TAX_YEAR_NOT_SUPPORTED" -> RuleTaxYearNotSupportedError
     )
 
-    errors ++ extraTysErrors
+    val hipErrors = Map(
+      "1215" -> NinoFormatError,
+      "1117" -> TaxYearFormatError,
+      "1217" -> EmploymentIdFormatError,
+      "1119" -> InternalError,
+      "1223" -> RuleCustomEmploymentUnignoreError,
+      "5010" -> NotFoundError,
+      "1115" -> RuleTaxYearNotEndedError,
+      "5000" -> RuleTaxYearNotSupportedError,
+      "4200" -> RuleOutsideAmendmentWindowError
```

Comment:
Please remove this error as RuleOutsideAmendmentWindowError was implemented only in V2.